streams/src/main/java/org/apache/kafka/streams/KeyQueryMetadata.java

```diff
@@ -71,11 +75,40 @@ public HostInfo getActiveHost() {
      * Get the Store partition corresponding to the key.
      *
      * @return store partition number
+     * @deprecated Use {@link #partition()} instead.
      */
+    @Deprecated
     public int getPartition() {
         return partition;
     }
 
+    /**
+     * Get the Active streams instance for given key
+     *
+     * @return active instance's {@link HostInfo}
+     */
+    public HostInfo activeHost() {
+        return activeHost;
+    }
+
+    /**
+     * Get the Streams instances that host the key as standbys
```

Comment:
nit: `Streams` -> `Kafka Streams`

Also, missing `.` at the end of the sentence.

Reply:
Fixed.